projects/scion/microfrontend-platform/package.json

```diff
@@ -20,7 +20,7 @@
   },
   "peerDependencies": {
     "rxjs": "^6.0.0",
-    "@scion/toolkit": "^10.0.0-beta.3 || ^10.0.0-rc.0 || ^10.0.0"
+    "@scion/toolkit": ">=11.0.0-beta.0"
```

Comment:
Please change the required toolkit version from `>=11.0.0-beta.0` to `>=10.0.0-beta.3` in order to not introduce a breaking change for projects using the platform without migrating to Angular 11. We do not require the toolkit@11.x version as no change is relevant to the microfrontend platform.

Reply:
done